generate/templates/api_doc.handlebars

```diff
@@ -26,7 +26,7 @@ import (
 	"log"
 	"os"
 
-	"github.com/hashicorp/vault-client-go"{{#each bodyParams}}{{#if @first}}
+	"github.com/hashicorp/vault-client-go"{{#each bodyParams}}{{#if isModel}}
 	"github.com/hashicorp/vault-client-go/schema"{{/if}}{{/each}}
```

Comment:
Explaining why this is safe: the bodyParams list is **guaranteed** to contain either 0 or 1 item - I have checked this in https://github.com/OpenAPITools/openapi-generator/blob/476c1a93972aca125d1021d448019661410736a7/modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java#L4588 - that line of code is **not** in a loop.

Thus we don't need to check for `@first` and moreover we already assumed it couldn't contain multiple items by naming every bodyParam literally `request` in the other template in this diff.